codalab_service.py

```diff
@@ -445,7 +447,7 @@ def has_callable_default(self):
         CodalabArg(
             name='worker_manager_{}_tag'.format(worker_manager_type),
             help='Tag of worker for {} jobs'.format(worker_manager_type),
-            default='codalab-{}'.format(worker_manager_type),
+            default='',
```

Comment:
Why remove this?

Reply:
I think otherwise you can't pass in an empty tag, or it will always use the default value (codalab-cpu or codalab-gpu).